hudi-common/src/main/java/org/apache/hudi/common/model/ActionType.java

```diff
@@ -22,5 +22,18 @@
  * The supported action types.
  */
 public enum ActionType {
-  commit, savepoint, compaction, clean, rollback, replacecommit, deltacommit, logcompaction, clustering
+  commit,
+  savepoint,
+  compaction,
+  clean,
+  rollback,
+  replacecommit,
+  deltacommit,
+  logcompaction,
+  clustering;
+
+  public static boolean isCommitActionType(String actionTypeStr) {
```

Comment:
can we add tests for this `isCommitActionType`

Reply:
I don't know how to test this, can you help with it?

Reply:
Can you help with it, I tried to add one in `TestColumnStatsIndex` but find it is hard to add explicit check for the index config.

If you meant to add a UT for the method, it is kind of unnecessary.